Editor/CKEditor/Resources/views/editorheader.html.twig

```diff
@@ -1,16 +1,21 @@
 <!-- start Scribite with CKEditor for {{ modname }} -->
 {# needs to be placed before the CKEditor files are included. #}
-{#<script type="text/javascript">#}
+{# <script type="text/javascript"> #}
     {# I'm not sure if a relative path or full path with schemeAndHost is better here #}
-    {#var CKEDITOR_BASEPATH = '{{ app.request.schemeAndHttpHost }}{{ asset('editors/ckeditor/ckeditor/') }}';#}
-{#</script>#}
+    {# var CKEDITOR_BASEPATH = '{{ app.request.schemeAndHttpHost }}/modules/Scribite/Editor/CKEditor/vendor/ckeditor/'; #}
+{# </script> #}
```

Comment:
Assets should be loaded from `web`. Scribite 6 cares for copying them automatically.